xwiki-platform-core/xwiki-platform-ratings/xwiki-platform-ratings-ui/src/main/resources/XWiki/RatingsTranslations.xml

```diff
@@ -42,8 +42,8 @@
   <content>admin.ratings=Ratings
 admin.ratings.activateratings=Activate the ratings on this wiki
 admin.ratings.activateratingsinfo=Activating the ratings on this wiki will display a visual average rating (composed of stars) at the top right corner of each document of this wiki (except the ones in the *XWiki* space). Authenticated users will be able to rate documents by clicking on the stars. An alternative to this is to use the *Ratings Panel*, for example to only enable ratings in the desired spaces. To use the Ratings Panel, please visit the *Panel Wizard* administration section.
-admin.ratings.resetratings=Reset all the ratings on this wiki will make all documents start over with no vote. Be aware that this operation is not reversible, besides on a document-per-document rollback basis. Resetting all ratings can take a while if the wiki has a big amount of documents already rated.
-admin.ratings.uninstallratings=Uninstalling the ratings application will delete the document that compose the application, but *all votes on rated document will be preserved*. If you reinstall the application later on, the documents will start over with the save average rating they had before uninstalling the application. Finally, uninstalling the ratings application will *remove the ratings category* from the wiki administration.
+admin.ratings.resetratings=Reset all the ratings on this wiki. This will make all documents start over with no vote. Be aware that this operation is irreversible, besides on a document-per-document rollback basis. Resetting all ratings can take a while if the wiki has a large amount of documents already rated.
+admin.ratings.uninstallratings=Uninstalling the ratings application will delete the document that composes the application, but *all votes on rated document will be preserved*. If you reinstall the application later on, the documents will start over with the save average rating they had before uninstalling the application. Finally, uninstalling the ratings application will *remove the ratings category* from the wiki administration.
```

Comment:
I believe the correct version would have been: "will delete the document`s` that compose the application".

Also, mind another small typo: "will start over with the save`d` average rating"

Reply:
Ok, Done!